open_instruct/grpo_fast.py

```diff
@@ -1360,7 +1453,7 @@ def main(args: Args, tc: TokenizerConfig, model_config: ModelConfig, reward_fn:
             dataset_local_cache_dir=args.dataset_local_cache_dir,
             dataset_skip_cache=args.dataset_skip_cache,
         )
-        eval_dataset = eval_dataset.shuffle(seed=args.seed)
+        # NOTE: eval dataset should not be shuffled
```

Comment:
Could we add a flag for this? Sometimes its nice to shuffle the eval dataset e.g. if the dataset is ordered in some way, and we want to observe performance on a random subset (to avoid large inference eval costs during training).

Reply:
Good point. Just added a flag.